packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardPreview.types.ts

```diff
@@ -43,6 +43,12 @@ export interface IDocumentCardPreviewProps extends IBaseProps<{}> {
    * Optional override class name
    */
   className?: string;
+
+  /**
+   * Maximum number of documents to display before adding the overflow text
+   * @DefaultValue 3
+   */
+  previewDocumentsToDisplayNumber?: number;
```

Comment:
`Number` seems redundant as it's already in the type. Could `maxDocumentCount` work?

Reply:
@cimontremblay I would agree; `maxDocumentDisplayCount` even.